README.md

```diff
@@ -298,6 +298,11 @@ The list of all the available styles can be obtained with (from GDB itself):
     python from pygments.styles import get_all_styles as styles
     python for s in styles(): print(s)
 
+To automatically cycle through and apply each available style (press return to try next style):
+
+    python from pygments.styles import get_all_styles as styles
+    python for s in styles(): gdb.execute('dashboard -style syntax_highlighting \'' + s + '\''); gdb.execute('dashboard'); print(s); input()
```

Comment:
`gdb.execute('dashboard');` is redundant, the dashboard redisplays automatically after a style change.

Also it would be nice to print the whole command for convenience, like this:

```py
python for s in styles(): c = 'dashboard -style syntax_highlighting {!r}'.format(s); gdb.execute(c); print(c); input()
```